
Remove some noise. With some C libraries the str* and bzero colide.

  - Rene Rebe <rene@exactcode.de>

diff -ur ash-linux-0.2/error.c ash-linux-0.2-fixed/error.c
--- ash-linux-0.2/error.c	1993-09-06 11:04:22.000000000 +0200
+++ ash-linux-0.2-fixed/error.c	2005-02-02 11:03:31.629060136 +0100
@@ -100,7 +100,7 @@
 		return;
 	}
 	intpending = 0;
-#ifdef BSD
+#ifdef BSD_NONO
 	sigsetmask(0);
 #endif
 	if (rootshell && iflag)
diff -ur ash-linux-0.2/mystring.h ash-linux-0.2-fixed/mystring.h
--- ash-linux-0.2/mystring.h	1993-08-24 11:12:53.000000000 +0200
+++ ash-linux-0.2-fixed/mystring.h	2005-02-02 11:01:35.411727856 +0100
@@ -44,14 +44,9 @@
 #ifdef __STDC__
 void scopyn(const char *, char *, int);
 char *strchr(const char *, int);
-void mybcopy(const pointer, pointer, int);
 int prefix(const char *, const char *);
 int number(const char *);
 int is_number(const char *);
-int strcmp(const char *, const char *);	/* from C library */
-char *strcpy(char *, const char *);	/* from C library */
-int strlen(const char *);		/* from C library */
-char *strcat(char *, const char *);	/* from C library */
 #else
 void scopyn();
 char *strchr();
@@ -67,4 +62,3 @@
 
 #define equal(s1, s2)	(strcmp(s1, s2) == 0)
 #define scopy(s1, s2)	((void)strcpy(s2, s1))
-#define bcopy(src, dst, n)	mybcopy((pointer)(src), (pointer)(dst), n)
